dotnet/.editorconfig

```diff
@@ -556,8 +552,8 @@ dotnet_diagnostic.IDE0060.severity = warning
 dotnet_diagnostic.IDE0062.severity = warning
 
 # IDE0073: File header
-dotnet_diagnostic.IDE0073.severity = suggestion
-file_header_template = Copyright (c) Microsoft. All rights reserved.
+dotnet_diagnostic.IDE0073.severity = warning
+file_header_template = Copyright (c) Microsoft Corporation. All rights reserved.\n{fileName}
```

Comment:
What is the motivation for having the filename in a comment? does it help Copilot?

Reply:
I just reuse the existing file_header_template to avoid making unnecessary updates to those files that already have a file header